packages/flutter_availability/lib/src/ui/widgets/template_time_selection.dart

```diff
@@ -27,46 +27,17 @@ class TemplateTimeSelection extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
```

Comment:
There are basically more variants of the TimeSelection in the component. Instead of passing in a variable title and description directly on the Screens that have this TimeSelection. I created 2 widgets that are slight variants of this TimeSelection. I could maybe move those widgets in the same file as the screens instead of seperate files but this way it keeps the code clean and readable(but with some more lines of code in total)

Reply:
TLDR i prefer the more structured way when there are slight variations of a widget